pkg/sfu/buffer/helpers.go

```diff
@@ -267,7 +269,7 @@ func IsH264Keyframe(payload []byte) bool {
 				return true
 			} else if n >= 24 {
 				// is this legal?
-				Logger.V(0).Info("Non-simple NALU within a STAP")
+				logger.Infow("Non-simple NALU within a STAP")
```

Comment:
using the generic logger in this one place. Was not worth jumping through hoops to get logger with context in here.